dot/core/service_test.go

```diff
@@ -325,9 +323,11 @@ func Test_Service_handleCodeSubstitution(t *testing.T) {
 			service := testCase.serviceBuilder(ctrl)
 
 			err := service.handleCodeSubstitution(testCase.blockHash, testCase.trieState)
-			assert.ErrorIs(t, err, testCase.errWrapped)
 			if testCase.errWrapped != nil {
-				assert.EqualError(t, err, testCase.errMessage)
+				assert.ErrorIs(t, err, testCase.errWrapped)
```

Comment:
you can revert to `assert.ErrorIs(t, err, testCase.errWrapped)` without the if guard